kafka-bin-scripts/README.adoc

```diff
@@ -94,6 +95,7 @@ To enable the Kafka scripts to access a Kafka instance, you must configure the c
 .Prerequisites
 ifndef::qs[]
 * You have the generated credentials for your service account. You copied this information previously for the Kafka instance in {product} by selecting the options menu (three vertical dots), clicking *Connection*, and creating the service account.
+* You've set the permissions for your service account to access the Kafka instance resources. You used the *Permissions* tab of the Kafka instance to set these permissions after you created the service account.
```

Comment:
So we do not recomend or going to test bin scripts acl cli? I find it odd to tell person to go to UI

Reply:
Same would apply to rhoas cli. If we want to get acl enabled before we add support we need the same note in rhoas cli tutorial
(after merge of #324 )

Reply:
Agreed. For now I've reworded in the fixes above to just suggest they use the UI for most ease. They can certainly use the CLI, but we've not yet documented ACLs in the CLI and it's not clear if it will make the external milestone.